python-sdk/examples/opencv/README.md

```diff
@@ -0,0 +1,16 @@
+# Distributed Image Processing with OpenCV and Python Pachyderm
+
+This is a reproduction of Pachyderm's opencv example in python. A walkthrough is available in the Pachyderm *[docs](https://docs.pachyderm.io/en/latest/getting_started/beginner_tutorial.html)*
+
+This example showcases the pachyderm-sdk analogs of common `pachctl` commands, such as creating repos and pipelines or getting a file.
+
+The image being used to run the pipeline code is built from the Dockerfile located at pachyderm/examples/opencv. You will also find the edges.py script there.
```

Comment:
[nit] I'd maybe provide a relative link within the repo or a url link to the pachyderm/example/opencv directory